src/vma/util/sys_vars.cpp

```diff
@@ -336,6 +336,41 @@ void mce_sys_var::read_env_variable_with_pid(char* mce_sys_name, size_t mce_sys_
 	}
 }
 
+bool mce_sys_var::check_cpuinfo_flag(const char* flag)
+{
+	bool ret = false;
+	char *line, *pch = NULL;
+	FILE *fp = NULL;
+	int line_size = MAX_CMD_LINE;
+
+	fp = fopen("/proc/cpuinfo", "r");
+	if (!fp) {
+		vlog_printf(VLOG_ERROR, "error while fopen\n");
+		print_vma_load_failure_msg();
+		exit(1);
+	}
```

Comment:
@serglelv please remove the following 2 lines (no reason to exit() if cpuinfo cannot be read)
```
print_vma_load_failure_msg();
exit(1);
```
Instead just return false
```
return false;
```